src/types.jl

```diff
@@ -98,6 +98,16 @@ end
 ###
 ### Term interface
 ###
+
+"""
+  symtype(x)
+
+Returns the symbolic type of `x`. By default this is just `typeof(x)`.
+Define this for your symbolic types if you want `SymbolicUtils.simplify` to apply rules
```

Comment:
```suggestion
    symtype(x)

Returns the numeric type of `x`. By default this is just `typeof(x)`.
Define this for your symbolic types if you want [`SymbolicUtils.simplify`](@ref) to apply rules
```

- A four-space indent is needed so that it is printed as Julia code.
See https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation
- Numeric type might be clearer. https://docs.julialang.org/en/v1/base/numbers/#Standard-Numeric-Types
- Add a cross-reference link to `SymbolicUtils.simplify`. https://documenter.juliadocs.org/stable/man/syntax/#@ref-link